### PR TITLE
build(tags): stop lerna from publishing extraneous git tags

### PR DESCRIPTION
**Pull request by redallen**
**What**: lerna currently adds extraneous git tags for patternfly3-react-lerna-root, patternfly4-react-lerna-root, and demo-app-ts. Deleting the versions from the respective package.json files stops `lerna-version` from publishing git tags thanks to this lerna PR: https://github.com/lerna/lerna/pull/1654

As a side effect, `yarn install` will no longer make separate node_modules folders in the following directories:
```
packages/patternfly-3/package.json
packages/patternfly-4/package.json
packages/patternfly-4/react-integration/demo-app-ts/package.json
```

**Additional Issues**: Because of the way yarn install works on version 1.7.0:
```
warning Missing version in workspace at "/Users/zallen/src/patternfly-react/packages/patternfly-3", ignoring.
warning Missing version in workspace at "/Users/zallen/src/patternfly-react/packages/patternfly-4", ignoring.
warning Missing version in workspace at "/Users/zallen/src/patternfly-react/packages/patternfly-4/react-integration/demo-app-ts", ignoring.
```
Their `devDependencies` need to be added to the root package.json as well. This is dirty, and yarn workspaces are one way to solve this in the future.

### CODE REVIEW

**Comment by redallen** [comment]
No idea why the jest Alert component tests are failing... kicking off another build: https://travis-ci.org/patternfly/patternfly-react/builds/490244913

**Comment by dgutride** [comment]
Alert tests are broken in master - this isn't your fault.

File: packages/patternfly-4/react-integration/package.json [hunk 1]
```diff
@@ -24,15 +24,15 @@
   "scripts": {
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
-    "integration:setup": "rimraf node_modules && yarn install",
+    "integration:setup": "shx rm -rf node_modules && yarn install",
```
Comment (redallen) [inline packages/patternfly-4/react-integration/package.json]: one less dependency!

File: package.json [hunk 2]
```diff
@@ -79,10 +118,10 @@
     "commitmsg": "commitlint -e",
     "coveralls": "shx cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "generate": "yarn plop",
-    "lerna": "lerna",
-    "lerna:publish": "lerna publish --yes",
+    "lerna:publish": "lerna publish --github-release --conventional-commits --yes",
```
Comment (redallen) [inline package.json]: See lerna PR: https://github.com/lerna/lerna/pull/1864

**Pull request merged**